### PR TITLE
[Docs] Japanese translation of docs/translating.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

Add Japanese translation of docs/translating.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/translating.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  ```js
+  fallbackLanguages: [
+    // ...
+    'zh-cn',
+    // ...
+  ],
+  ```
+
+## Previewing the Translations
```
Comment (shelaf) [inline docs/ja/translating.md:56]:
```suggestion
## 翻訳のプレビュー
```

File: docs/ja/translating.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# 翻訳を追加する方法
+
+<!---
+  original document: 0.9.51:docs/translating.md
+  git diff 0.9.51 HEAD -- docs/translating.md | cat
+-->
+
+ルートフォルダ (`docs/`) にある全てのファイルは英語でなければなりません - 他の全ての言語は、ISO 639-1 言語コードと、それに続く`-`と適切な国コードのサブフォルダにある必要があります。[一般的なもののリストはここで見つかります](https://www.andiamo.co.uk/resources/iso-language-codes/)。このフォルダが存在しない場合、作成することができます。翻訳された各ファイルは英語バージョンと同じ名前でなければなりません。そうすれば正常に元の状態に戻すことができます。
```
[the remark this answers is not in the document]
Comment (umi-umi) [reply docs/ja/translating.md]: ご指摘ありがとうございます。
どこから「適切な」なんて引っ張ってきたんだろう。。

**Comment by noroadsleft** [comment]
@mtei Go ahead and merge this when you feel it is ready.

File: docs/ja/translating.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# QMK ドキュメントを翻訳する
+
+<!---
+  original document: 0.9.51:docs/translating.md
+  git diff 0.9.51 HEAD -- docs/translating.md | cat
+-->
+
+ルートフォルダ (`docs/`) にある全てのファイルは英語でなければなりません - 他の全ての言語は、ISO 639-1 言語コードと、それに続く`-`と関連する国コードのサブフォルダにある必要があります。[一般的なもののリストはここで見つかります](https://www.andiamo.co.uk/resources/iso-language-codes/)。このフォルダが存在しない場合、作成することができます。翻訳された各ファイルは英語バージョンと同じ名前でなければなりません。そうすれば正常に元の状態に戻すことができます。
```
Comment (mtei) [inline docs/ja/translating.md:8]:
```suggestion
ルートフォルダ (`docs/`) にある全てのファイルは英語でなければなりません - 他の全ての言語は、ISO 639-1 言語コードと、それに続く`-`と関連する国コードのサブフォルダにある必要があります。[一般的なもののリストはここで見つかります](https://www.andiamo.co.uk/resources/iso-language-codes/)。このフォルダが存在しない場合、作成することができます。翻訳された各ファイルは英語バージョンと同じ名前でなければなりません。そうすることで、正常にフォールバックできます。
```

**Review by mtei — APPROVED** [review]

**Comment by mtei** [comment]
Thank you!

**Pull request merged**